source/src_lcao/center2_orb-orb11.cpp

```diff
@@ -151,3 +151,92 @@ double Center2_Orb::Orb11::cal_overlap(
 	
 	return overlap;
 }
+
+ModuleBase::Vector3<double> Center2_Orb::Orb11::cal_grad_overlap(    //caoyu add 2021-11-19
+	const ModuleBase::Vector3<double> &RA, const ModuleBase::Vector3<double> &RB,
+    const int& mA, const int& mB) const
+{
+    const double tiny1 = 1e-12;		// why 1e-12?
```

Comment:
Yes, why we choose this standard? You might figure it out with someone in charge.